app/i18n/en/index.ts

```diff
@@ -842,7 +842,7 @@ const en: BaseTranslation = {
     learn: "I don't mean to badger you, but there's lot more to learn, dig in...",
     learnToEarn: "Learn to Earn",
   },
-  PhoneInputScreen: {
+  PhoneLoginInitiateScreen: {
```

Comment:
In may make sense to unify `PhoneLoginInitiateScreen` and `PhoneRegistrationInitiateScreen` to `PhoneAuthInitiateScreen` just because they seem to be almost identical. Same with `PhoneLoginValidationScreen` and `PhoneRegistrationValidationScreen`.